libandroid-navigation-ui/src/test/java/com/mapbox/services/android/navigation/ui/v5/BaseTest.java

```diff
@@ -27,4 +33,12 @@ protected String loadJsonFixture(String filename) throws IOException {
     Scanner scanner = new Scanner(inputStream, UTF_8.name()).useDelimiter("\\A");
     return scanner.hasNext() ? scanner.next() : "";
   }
+
+  protected LegStep getFirstStep(DirectionsRoute route) {
```

Comment:
Is there somewhere else these could go? It seems very specific to be included in all tests